src/dataloader/dataloader.py

```diff
@@ -57,8 +57,13 @@ def __init__(self, data_path, total_image_num, train=True):
             now selecting randomly generated n number of images. n is mentioned in total_images in params.json
             mulitplier 0.3 mentioned to make sure we select 0.3 * n images from all testing images
             '''
-            self.dataset = random_image_selection(
```

Comment:
Why is there if elif inside else? We can probably have just if elif else

Reply:
If the dataset is not train then there are less transformations for val and test images. They are all contained in the else block, so it made more sense to me to have it there instead of having an if elif else block, just a shorter code version

Reply:
We need the code to be understandable so having more lines of code is preferred if it makes it more readable.

Reply:
I've added comments for it. If needed, I can add more lines of code. Let me know once you look at it.

Reply:
If this looks good @vishnu-m77 can you resolve the comment?